flow/designs/tsmc65lp/swerv_wrapper/config_hier.mk

```diff
@@ -0,0 +1,43 @@
+export DESIGN_NAME = swerv_wrapper
+export PLATFORM    = tsmc65lp
+
+#export FLOW_VARIANT = hier
+export FLOW_VARIANT = hier_rtlmp
+export SYNTH_HIERARCHICAL = 1
+export MAX_UNGROUP_SIZE = 100
+export RTLMP_FLOW = True
+
+export VERILOG_FILES = ./designs/src/swerv/swerv_wrapper.sv2v.v \
+                       ./designs/$(PLATFORM)/$(DESIGN_NAME)/macros.v
+export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NAME)/constraint.sdc
+
+export ADDITIONAL_LEFS = $(PLATFORM_DIR)/lef/tsmc65lp_1rf_lg11_w40_all.lef \
+                         $(PLATFORM_DIR)/lef/tsmc65lp_1rf_lg6_w22_all.lef \
+                         $(PLATFORM_DIR)/lef/tsmc65lp_1rf_lg8_w34_all.lef
+export ADDITIONAL_LIBS = $(PLATFORM_DIR)/lib/tsmc65lp_1rf_lg11_w40_all_ss_1p08v_1p08v_125c.lib \
+                         $(PLATFORM_DIR)/lib/tsmc65lp_1rf_lg6_w22_all_ss_1p08v_1p08v_125c.lib \
+                         $(PLATFORM_DIR)/lib/tsmc65lp_1rf_lg8_w34_all_ss_1p08v_1p08v_125c.lib
+export ADDITIONAL_GDS  = $(PLATFORM_DIR)/gds/tsmc65lp_1rf_lg11_w40_all.gds2 \
+                         $(PLATFORM_DIR)/gds/tsmc65lp_1rf_lg6_w22_all.gds2 \
+                         $(PLATFORM_DIR)/gds/tsmc65lp_1rf_lg8_w34_all.gds2
+
+export FLOORPLAN_DEF = ./results/$(PLATFORM)/$(DESIGN_NAME)/$(FLOW_VARIANT)/2_2_floorplan_io.def
+#
+# RTL_MP Settings
+export RTLMP_MAX_INST = 5000
+export RTLMP_MIN_INST = 1000
+export RTLMP_MAX_MACRO = 12
+export RTLMP_MIN_MACRO = 4
+export RTLMP_RPT_DIR = $(OBJECTS_DIR)/rtlmp
+export RTLMP_RPT_FILE = partition.txt
+export RTLMP_CONFIG_FILE = ./designs/$(PLATFORM)/$(DESIGN_NAME)/rtlmp_config.txt
+export RTLMP_BLOCKAGE_FILE = ./designs/$(PLATFORM)/$(DESIGN_NAME)/macro_blockage.txt
+
+export REMOVE_BUFFER_TREE     = 1
+
+export DIE_AREA    = 0 0 1610 1410
+export CORE_AREA   = 5 5 1605 1405
+export PLACE_PINS_ARGS = -exclude left:* -exclude right:* -exclude top:* -exclude bottom:0-300 -exclude bottom:1300-1600
+
+export PLACE_DENSITY ?= .60
```

Comment:
Does this line actually do anything? The `PLACE_DENSITY` should be set by the platform already so it won't get overwritten with `?=`

Reply:
Changed to '='  However the platform config is read after the design config files currently.